client/my-sites/plans-features-main/gutenboarding-header/style.scss

```diff
@@ -1,18 +1,37 @@
 .gutenboarding-header {
+
+	// START - exact styles from my-sites/plan-features/style.scss targeting .is-section-signup .plans-features-main__group.is-scrollable
+	width: 100vw;
+	margin-left: calc( 50% - 50vw );
+
+	@media ( min-width: 1600px ) {
+		max-width: 1600px;
+		margin-left: -280px;
+	}
+
+	@include breakpoint-deprecated( '<1040px' ) {
+		padding-top: 12px;
+	}
+	// END
+
```

Comment:
These changes are a bit hacky and are not really related to the main issue being flagged here — not sure if it's best to include them (and add a follow-up issue/PR to fix those styles properly), or omit these changes completely.

Reply:
If we want to [move the header above toggle](https://github.com/Automattic/wp-calypso/pull/48811/files#diff-2545c62593dd002397248377b1c0feb643bd2acc7694054ce094997b1c029aceR432) and keep it aligned with plans grid, hacky is the only way 😬